sdk/storage/AzureStorageBlob/Source/Data/BlobTransfer+CoreDataClass.swift

```diff
@@ -85,6 +85,8 @@ public class BlobTransfer: NSManagedObject, Transfer {
         }
     }
 
+    internal var previousState: TransferState?
```

Comment:
I avoided adding this previously because I didn't want to serialize it... however, unlike the property for previous progress, there really is no need to save this, so I just track it internally.